assets/tailwind.config.js

```diff
@@ -102,6 +102,10 @@ module.exports = {
         },
         "brand-pink": "#e44c75",
       },
+      gridTemplateColumns: {
+        'apps': 'minmax(0, 2fr) minmax(0, 2fr) minmax(0, 1fr) minmax(0, 1fr) minmax(0, 1fr)',
+        'apps-m': 'minmax(0, 0.5fr) minmax(0, 0.75fr) minmax(0, 0.5fr) minmax(0, 0.5fr) minmax(0, 0.5fr)',
```

Comment:
I would avoid adding too much global customizations, especially that this one is definitely apps specific. I believe you can use it inline like `grid-cols-[minmax(0,_2fr)_minmax(0,_2fr)_minmax(0,_1fr)_minmax(0,_1fr)_minmax(0,_1fr)]` (not pretty at least not global :)).